src/xml/percolator_out.xsd

```diff
@@ -96,7 +96,7 @@ elementFormDefault="qualified" attributeFormDefault="unqualified">
   </xsd:annotation>
 </xsd:element>
 
- <xsd:element name="calc_mass" type="xsd:double">
```

Comment:
Please revert this edit (it is just white space, but this file must not be changed)

Reply:
Fixed ef7b92c.